src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -531,13 +550,14 @@ default FieldDefinition.LookupInfo getLookupInfo()
             return null;
         }
 
-        /**
-         * @deprecated Bridge for converting away from enum
-         */
-        @Deprecated (since = "22.10")
```

Comment:
I'd previously thought it would go away eventually but `DomainFieldRow.getType` uses it. We could refactor `DomainFieldRow.getType` to return a `String` instead but that doesn't seem like a great option.